core/src/main/java/nl/nn/adapterframework/doc/FrankMandatory.java

```diff
@@ -0,0 +1,25 @@
+/*
+   Copyright 2021 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.doc;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FrankMandatory {
```

Comment:
Ik denk dat het mooier is als we puur Mandatory gebruiken als java annotatie, en de prefix `ff.` gebruiken als javadoc annotatie. Dan kunnen we dit ook voor default en protected toe gaan passen.